### DISCUSSION TITLE
PDF 1.7 (ISO) generation

### DISCUSSION THREAD

**Discussion by Sebazzz** [QuestPDF/QuestPDF#633] [Q&A] [answered]
Is there any way to generate PDFs that comply with the ISO standard (1.7 or higher)? Currently QuestPDF appears to generate PDF v1.4 PDFs only.

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We apologize for the long delay. Since this thread was opened, QuestPDF has added support for ISO-standardized PDF output. You can now generate PDF/A (ISO 19005, archival) and PDF/UA (ISO 14289, accessibility) conformant documents through the PDFA_Conformance and PDFUA_Conformance options in DocumentSettings. As PDF/A-2 and PDF/A-3 are built on the ISO 32000-1 (PDF 1.7) base, enabling conformance produces documents that meet these formal standards.

You can find the details here:
https://www.questpdf.com/concepts/document-settings.html
https://www.questpdf.com/concepts/accessibility.html

We are closing this thread as resolved, but should you have any further questions, please don't hesitate to open a new issue.

Thank you!
